global_schema.ddl

```diff
@@ -120,6 +90,32 @@ CREATE TABLE IF NOT EXISTS controller (
 		REFERENCES	cloud(uuid)
 )
 
+CREATE TABLE IF NOT EXISTS api_host_port (
+	uuid			TEXT PRIMARY KEY,
+	value			TEXT,
+	address_type	TEXT,
+	scope			TEXT,
+	port			INT,
+	space_id		TEXT, -- TODO: Investigate this field
+	controller_uuid	TEXT NOT NULL,
+	CONSTRAINT		fk_api_host_port_controller
+		FOREIGN KEY	(controller_uuid)
+		REFERENCES	controller(uuid)
+);
+
+CREATE TABLE IF NOT EXISTS api_host_port_for_agents (
```

Comment:
The controller address/port (L4) entries usable by agents will be a subset of those usable by clients (all of them). So let's drop this table and just add a boolean `agent_usable` column to the table above.